packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -426,6 +426,13 @@ const double _kDefaultFontSize = 14.0;
 ///
 /// ### Inconsistent platform fonts
 ///
+/// By default, fonts differ depending on the platform. 
+///
+///  * Default font-family for `Android`,`Fuchsia` and `Linux` is `Segoe UI`.
```

Comment:
isn't the default font for android Roboto?

Reply:
Oh, my bad.

Reply:
Yes. My bad